### ISSUE TITLE
Wazuh API error: 2002 easily triggered.

### ISSUE THREAD

**Issue by jctello** [wazuh/wazuh#1943]
## Bug report

**OS**
CentOS7

**Wazuh version**
3.7.0

**Wazuh API version**
3.7.0

**Install type**
Manager with one agent

**Install method**
packages (from repository)

**Log sample**
`/var/ossec/logs/api.log`: 
```
WazuhAPI 2018-11-23 14:00:25 foo: [::ffff:127.0.0.1] GET /rootcheck/001/last_scan? - 200 - error: '2002'.
```

Bug description
----------------
Repeating the process of loading an Agent's information from the Agent's tab of the Wazuh Kibana App and going back to the Agents tab 3-5 consecutive times will issue the message:
```
Agents. Wazuh API error: 2002 - Maximum attempts exceeded for sqlite3 execute: database is locked
```
![image](https://user-images.githubusercontent.com/36071202/48954765-38d6ff80-ef4b-11e8-869c-6e16cf1b04fa.png)
And then get stuck on loading until another tab is clicked on again.

**Comment by jesuslinares** [comment]
Hi @jctello,

We are aware of this error. It wil be fixed when we use _wazuh-db_ to retrieve the rootcheck information. More details here: https://github.com/wazuh/wazuh/issues/742.

Thanks.

**Comment by crd1985** [comment]
Fixed in #2869
